modules/S28_python_run/embamodule.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python3
+"""
+EMBA - EMBEDDED LINUX ANALYZER
+
+Copyright 2024-2024 Siemens Energy AG
```

Comment:
Same here

I would suggest usage of __author__
And __copyright__ but it's up to you